onnxconverter_common/oopb.py

```diff
@@ -332,8 +332,8 @@ def reciprocal(self, inputs, name=None, outputs=None):
     def relu(self, inputs, name=None, outputs=None):
         return self.apply_op(onnx_ops.apply_relu, inputs, name, outputs)
 
-    def relu6(self, inputs, name=None, outputs=None, dtype=None):
-        return self.apply_op(onnx_ops.apply_relu6, inputs, name, outputs, dtype=dtype)
+    def relu_6(self, inputs, name=None, outputs=None, zero_value=0.0):
```

Comment:
Do we need manually run code_gen after each `onnx_ops` change?

Reply:
yes.